package.json

```diff
@@ -1,60 +1,51 @@
 {
   "name": "draxo-react-timer",
-  "version": "1.0.23",
+  "version": "1.0.24",
   "description": "Draxo React Timer",
-  "private": false,
-  "main": "src/ReactTimer.tsx",
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+  "private": true,
+  "type": "module",
+  "repository": {
+    "url": "https://github.com/draxoco/draxo-react-timer",
+    "type": "github"
   },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
+  "files": [
+    "dist",
+    "!dist/App.js",
+    "!dist/main.js"
+  ],
+  "release": {
+    "branches": [
+      "main"
     ]
   },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
```

Comment:
me podrías explicar la ubicación de este archivo o como esta configurado?

Reply:
Está es la ubicación en desarrollo -> https://github.com/jorgeart81/draxo-react-timer/blob/feature/no-ref/migrate-to-vite/src/index.ts
Y como biblioteca esto está en dentro de la carpeta dist que se genera con el build. Este index da acceso al hook del timer.